app/views/case_contacts/_case_contact.html.erb

```diff
@@ -13,15 +13,15 @@
               <br>
               <%= contact.decorate.subheading %>
             </h6>
-            <div class="card-text">
+            <div class="card-text js-read-more-text-wrapper">
```

Comment:
Couldn't come up with a better name for this one. Let me know if you have any suggestions.